src/sections/2021/section-2.js

```diff
@@ -0,0 +1,176 @@
+import React from "react"
+import SectionHeader from '../../components/2021/section-header'
+import Grid from '../../components/2021/grid'
+import GridCell from '../../components/2021/grid-cell'
+import PieChart from '../../components/2021/pie-chart'
+import BarChart from '../../components/2021/bar-chart'
+import Quote from '../../components/2021/quote'
+import Figure from "../../components/2021/figure"
+import SmallCallout from "../../components/2021/small-callout"
+
+const Section2 = () => (
+    <Grid>
+      <GridCell spanMD="8" spanLG="12">
+        <SectionHeader title="Design System & Their Teams" number="2" total="6" />
+      </GridCell>
+    
+      <GridCell>
+        <p>The elements in your design system and the team creating them are foundational to building a quality product. Let’s learn more about the commonalities across our respondents’ design systems.</p>
+      </GridCell>
+      
+      <GridCell>
+        <Figure count="1.1" caption="159 Responses">
+          <PieChart
+            styleFormat="large"
+            title="Almost 40% of systems were successful or very successful"
+            dataPoints={[
+              ['Not successful', 4],
+              ['Slightly successful', 19],
+              ['Moderately successful', 38],
+              ['Successful', 31],
+              ['Very successful', 8]]
+          }>
+            <p>In your opinion, how <em>successful</em> is your organization’s design system?</p>
+          </PieChart>
+        </Figure>
+      </GridCell>
+      
+      <GridCell>
+        <p>Most in-house respondents felt that their design system is either moderately successful (38%) or successful to very successful (39%).</p>
+      </GridCell>
+      
+      <GridCell>
+        <Quote cite="Executive, successful design system">
+          <p>It has not only outperformed in terms of KPIs, but it has also allowed us to scale more work with fewer people.</p>
```

Comment:
Add the large body copy class here and `<em><strong>` tags around the bold text from the doc.

Reply:
The large body copy will be addressed in #260.